packages/venia-ui/lib/components/App/app.js

```diff
@@ -35,21 +34,17 @@ const App = props => {
 
     const [{ toasts }, { addToast }] = useToasts();
 
-    const [, appApi] = useAppContext();
-
-    const {
-        actions: { htmlUpdateAvailable }
-    } = appApi;
+    const [htmlUpdateAvailable, setHTMLUpdateAvailable] = useState(false);
```

Comment:
Yay! Thanks for switching this. That said, this state should be created in the `talon`. Just move this over and have `htmlUpdateAvailable` and `setHTMLUpdateAvailable` as return props.

Also, the naming is weird with `html` and `HTML`. If you change it to `isUpdateAvailable and `setIsUpdateAvailable` it would match the hook naming conventions...I think.

Never ran into casing issues with this before :P

Reply:
Done 👍